penguin/security/tool_permissions.py

```
@@ -10,7 +10,10 @@
 import logging
 import re
 from pathlib import Path
-from typing import Any, Dict, List, Optional, Tuple
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple
```

Ruff (0.15.12)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)